frontend/js/src/common/brainzplayer/BrainzPlayer.tsx

```diff
@@ -220,7 +218,38 @@ export default class BrainzPlayer extends React.Component<
       // Received postMessage from different origin, ignoring it
       return;
     }
+    const { userPreferences } = this.context;
     const { brainzplayer_event, payload } = event.data;
+    if (!brainzplayer_event) {
+      return;
+    }
+    if (userPreferences?.brainzplayer) {
+      const brainzPlayerDisabled =
+        userPreferences?.brainzplayer?.spotifyEnabled === false &&
+        userPreferences?.brainzplayer?.youtubeEnabled === false &&
+        userPreferences?.brainzplayer?.soundcloudEnabled === false;
+      if (brainzPlayerDisabled) {
+        toast.info(
+          <ToastMsg
+            title="BrainzPlayer disabled"
+            message={
+              <>
+                You have disabled all music services for playback on
+                ListenBrainz. To enable them again, please go to the{" "}
+                <a
+                  href="/settings/brainzplayer/"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  music player preferences
+                </a>{" "}
+                page
+              </>
+            }
+          />
+        );
+      }
```

Comment:
If brainzplayer is disabled, the toast is displayed twice
```suggestion
        return;
```